Rebuild All in VS generates failures

Doing Rebuild All gives errors:
```
14>LINK : fatal error LNK1104: cannot open file 'C:\git\dthaler\ebpf-for-windows\external\ebpf-verifier\build\Debug\ebpfverifier.lib'
14>Done building project "ebpfapi.vcxproj" -- FAILED.
17>LINK : fatal error LNK1104: cannot open file 'C:\git\dthaler\ebpf-for-windows\x64\Debug\EbpfApi.lib'
17>Done building project "port_quota.vcxproj" -- FAILED.
16>LINK : fatal error LNK1104: cannot open file 'C:\git\dthaler\ebpf-for-windows\x64\Debug\EbpfApi.lib'
16>Done building project "ebpfnetsh.vcxproj" -- FAILED.
15>LINK : fatal error LNK1104: cannot open file 'C:\git\dthaler\ebpf-for-windows\external\ebpf-verifier\build\Debug\ebpfverifier.lib'
15>Done building project "end_to_end.vcxproj" -- FAILED.
```
But building again after that works fine, so must be a dependency/ordering problem.

--- a/src/ebpf/dll/ebpfapi.vcxproj
+++ b/src/ebpf/dll/ebpfapi.vcxproj
@@ -126,7 +126,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\external\ebpf-verifier\build\ebpfverifier.vcxproj">
-      <Project>{bd587773-916f-365f-82c0-2e8db12f40a0}</Project>
+      <Project>{7d5b4e68-c0fa-3f86-9405-f6400219b440}</Project>
     </ProjectReference>
     <ProjectReference Include="..\libs\api\api.vcxproj">
       <Project>{c8bf60c3-40a9-43ad-891a-8aa34f1c3a68}</Project>

--- a/src/test/end_to_end/end_to_end.vcxproj
+++ b/src/test/end_to_end/end_to_end.vcxproj
@@ -107,7 +107,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\external\ebpf-verifier\build\ebpfverifier.vcxproj">
-      <Project>{bd587773-916f-365f-82c0-2e8db12f40a0}</Project>
+      <Project>{7d5b4e68-c0fa-3f86-9405-f6400219b440}</Project>
     </ProjectReference>
     <ProjectReference Include="..\..\ebpf\libs\api\api.vcxproj">
       <Project>{c8bf60c3-40a9-43ad-891a-8aa34f1c3a68}</Project>

